lib/ContextParser.ts

```diff
@@ -632,18 +631,21 @@ must be one of ${Util.CONTAINERS.join(', ')}`, ERROR_CODES.INVALID_CONTAINER_MAP
    * @param {IParseOptions} options Optional parsing options.
    * @return {Promise<JsonLdContextNormalized>} A promise resolving to the context.
    */
+  public async parse(context: JsonLdContext, options?: IParseOptions): Promise<JsonLdContextNormalized>
   public async parse(context: JsonLdContext,
-                     options: IParseOptions = {}): Promise<JsonLdContextNormalized> {
+                     options: IParseOptions = {},
+                     // These options are only for internal use on recursive calls and should not be used by
```

Comment:
The bottom type definition is internal to the library whilst all type overloads above it are shown to consumers. This means that consumers see the type of this function as ` public async parse(context: JsonLdContext, options?: IParseOptions): Promise<JsonLdContextNormalized>`.

This means this comment does not need to be added to the tsdoc - because consumers do not see this argument in the first place.